include/seqan3/alignment/pairwise/policy/find_optimum_policy.hpp

```diff
@@ -66,6 +74,18 @@ class find_optimum_policy
     constexpr find_optimum_policy & operator=(find_optimum_policy const &) = default; //!< Defaulted
     constexpr find_optimum_policy & operator=(find_optimum_policy &&) = default;      //!< Defaulted
     ~find_optimum_policy() = default;                                                 //!< Defaulted
+
+    //!\brief Initialises the policy with the configuration.
+    template <typename configuration_t>
+    find_optimum_policy(configuration_t const & config)
+    {
+        if constexpr (configuration_t::template exists<method_local_tag>())
+            test_every_cell =  true;
+
+        auto method_global_config = config.get_or(align_cfg::method_global{});
+        test_last_row_cell = method_global_config.free_end_gaps_sequence1_trailing;
+        test_last_column_cell = method_global_config.free_end_gaps_sequence2_trailing;
```

Comment:
aren't `test_every_cell` and `test_last_row/column_cell` contradictory ?
So why set both and not just in an else branch ?

Reply:
Not necessarily. But the functions to check for a new score are implemented sucht that if every cell is tracked the explicit functions for tracking the last row/column won't check the same cell a second time.